modules/ossm-rn-new-features.adoc

```diff
@@ -103,6 +125,28 @@ Additionally, the SMMR must also be configured for cluster-wide deployment. This
 ----
 <1> Adds all namespaces to the mesh, including any namespaces you subsequently create. The following namespaces are not part of the mesh: kube, openshift, kube-* and openshift-*.
 
+== New features {SMProductName} version 2.2.5
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on OpenShift Container Platform 4.9 or later.
```

Comment:
Replace instances of "OpenShift Container Platform" with the `{product-title}` attribute.

Reply:
done. good point.